src/reducers/TodoReducer.js

```diff
@@ -0,0 +1,62 @@
+import constants from "../constants";
+
+const initialState = {
+  todos: [],
+  totalCount: null,
+  pagesCount: null,
+};
+
+const { GET_TODOS, ADD_TODO, UPDATE_TODO, UPDATE_TODOS, TODOS_AT_PAGE } =
```

Comment:
I think we can dont use destruct and use it directly - CamelCaseName.GET_TODOS, etc..